_news/20230828-papers.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: news
+title: New tskit-related papers
+description: | 
+  A trio of papers in the prestigious journals Nature, Science, and Nature Genetics
+  illustrate the power of tskit-based simulation and analysis to carry out
+  genome-wide analysis of human genome data. These focus on deep human ancestry,
+  recent human pedigrees, and statistical algorithms for biobank-scale datasets.
+image: ../assets/images/news/20230828-papers/quebec-thin.jpg
+timestamp: 2023-08-28
+author: Yan Wong
+---
+
+# A trio of high-impact papers
+
+In May and August 2023 the following papers, which make extensive use of tskit projects, were published:
+
+#### A weakly structured stem for human origins in Africa – [<span class="link__text">Nature</span>](https://www.nature.com/articles/s41586-023-06055-y){:class="link--type-1--external"}
+{: style="clear:both"}
+
+![Figure 3b: demographic plot of a weak stem with multiple mergers]({{ '/assets/images/news/20230828-papers/human-stem.jpg' |  relative_url}}){:class="news-float-left" style="width: 12em;"}
+
+A paper by [@apragsdale](https://github.com/apragsdale) and colleagues, who contribute to
```

Comment:
I think we'd want to get approval from @apragsdale on this. We can't really put out a news item for every paper that uses msprime, or even every such paper that is written by an msprime developer.

Reply:
I'm perfectly happy to have this included here.

msprime usage is really high of course, especially among developers, so that is a reasonable point Jerome.

Reply:
Yeah, I would only really put something out like this if it's a Nature / Science / etc paper that had involvement of some of the core tskit / msprime / stdpopsim devs.

Reply:
It's worth showing, I think, that the `tskit` community has members who are actively publishing high profile papers.

Reply:
All good then, I just wanted to be sure Aaron was happy!